apps/server/src/modules/management/uc/database-management.uc.ts

```diff
@@ -1,15 +1,17 @@
 /* eslint-disable no-await-in-loop */
```

Comment:
Not your code, but can you please add it to each line instead of a eslint-disable for the whole file?